modules/twinkleblock.js

```diff
@@ -1040,6 +1040,14 @@ Twinkle.block.blockPresetsInfo = {
 		reasonParam: true,
 		reason: 'Email [[WP:Harassment|harassment]]',
 		summary: 'You have been [[WP:PB|blocked]] from emailing other editors for [[WP:Harassment|harassment]]'
+	},
+	'uw-aepblock': {
```

Comment:
`forRegisteredOnly: true,`

Reply:
Why?  `uw-aeblock` isn't

Reply:
> Why? `uw-aeblock` isn't
Not too sure about this? Do we issue AE blocks to IPs? 
I assumed the system goes like:
* Page under sanction
* Editor (not IP alerted)
* Blocked for violating sanction

Reply:
It's rare, I imagine largely because of the extra paperwork and notice required, and because things like `uw-disruptblock` are just easier, but it does happen and there's no reason why not.  DS applies to editors, not just registered users.